homeassistant/components/tahoma/__init__.py

```diff
@@ -33,6 +33,7 @@
 TAHOMA_COMPONENTS = ["scene", "sensor", "cover", "switch", "binary_sensor"]
 
 TAHOMA_TYPES = {
+    "io:VerticalInteriorBlindVeluxIOComponent": "cover",
```

Comment:
It looks like this list is alphabetical ordered. Please add this line after `"io:VerticalExteriorAwningIOComponent": "cover",`